.github/workflows/ci.yml

```
@@ -155,9 +161,10 @@ jobs:
         with:
           enable-cache: true
           version: ${{ env.UV_VERSION }}
+          version: ${{ env.UV_VERSION }}
```

actionlint (1.7.9)

key "version" is duplicated in "with" section. previously defined at line:163,col:11. note that this key is case insensitive

(syntax-check)